demo-app/src/open-truss/configs/7-render-frames-demo.yaml

```diff
@@ -0,0 +1,62 @@
+workflow:
+  version: 1
+  id: 7-render-frames-demo-unique-id
+  signals:
+    integer: number
+    runCalculation: NavigateFrame
+  renderFrames:
+    type: inSequence
+    next: :runCalculation # sets the next function as :runCalculation
```

Comment:
Been thinking about the naming of things a bit and wonder if shortening `renderFrames` to `render` and changing `type: inSequence` to `mode: sequential` reads a bit better 🤔 

```
  render:
    mode: sequential
    next: :runCalculation
```

The default mode which could be omitted:
```
  render:
    mode: parallel
```

Reply:
Interesting wrinkle is `renderFrames` explicitly applies only to how we render the sub frames if they exist. If we ever need an annotation that determines how to render the frame itself then we'd need something like `renderFrame`. 

Perhaps we can run with `renderFrames` for now and after a while if that seems sufficient we can rename to `render`?